### PR TITLE
Root Block Device Encryption Policy

### PR DESCRIPTION
**Pull request by andrewpurdin**
* Add validation for encryption of root block devices to existing `encrypted-volumes` policy
* Add unit tests related to `encrypted-volumes` policy
* Add Integration tests to validate root block device encryption

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +0,0 @@
-## 0.2.1 (2020-05-29)
```
Comment (justinvp) [inline CHANGELOG.md:1]: Please add back this file, and add a `## HEAD (Unreleased)` section at the top with a bullet that summarizes this change.

File: src/tests/compute.spec.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import { ResourceValidationArgs } from "@pulumi/policy";
+
+
+import * as compute from "../compute";
+
+import { assertHasResourceViolation, assertNoResourceViolations, createResourceValidationArgs } from "./util";
+
+
+describe("#ec2BlockDeviceEncryption", () => {
+  const policy = compute.encryptedVolumes;
```
Comment (justinvp) [inline src/tests/compute.spec.ts:27]: Nit: Use 4 spaces for indentation rather than 2.

File: CHANGELOG.md (hunk 1 above)
Comment (andrewpurdin) [reply CHANGELOG.md:1]: How's this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,18 @@
+## HEAD (Unreleased)
+
+- Add root volume encryption validation to `encrypted-volumes` policy.
+
 ## 0.2.1 (2020-05-29)
 
-- Fix TypeScript compiler error when using the library (thanks @rsclarke-vgw!)
-  [#60](https://github.com/pulumi/pulumi-policy-aws/pull/60)
+- Fix TypeScript compiler error when using the library (thanks @rsclarke-vgw!)	
+  [#60](https://github.com/pulumi/pulumi-policy-aws/pull/60)
```
Comment (justinvp) [inline CHANGELOG.md:8]:
```suggestion
- Fix TypeScript compiler error when using the library (thanks @rsclarke-vgw!)
  [#60](https://github.com/pulumi/pulumi-policy-aws/pull/60)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
+
 ## 0.2.1 (2020-05-29)
 
-- Fix TypeScript compiler error when using the library (thanks @rsclarke-vgw!)
-  [#60](https://github.com/pulumi/pulumi-policy-aws/pull/60)
+- Fix TypeScript compiler error when using the library (thanks @rsclarke-vgw!)	
+  [#60](https://github.com/pulumi/pulumi-policy-aws/pull/60)	
 
-## 0.2.0 (2020-04-17)
+## 0.2.0 (2020-04-17)
```
Comment (justinvp) [inline CHANGELOG.md:10]:
```suggestion
## 0.2.0 (2020-04-17)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,14 +1,18 @@
...
 
-## 0.2.0 (2020-04-17)
+## 0.2.0 (2020-04-17)	
 
-- Use consistent naming of policies.
-- Add rdsInstanceMultiAZEnabled policy.
-- Upgrade to @pulumi/policy v1.1.0 and @pulumi/pulumi v2.0.0.
+- Use consistent naming of policies.	
+- Add rdsInstanceMultiAZEnabled policy.	
+- Upgrade to @pulumi/policy v1.1.0 and @pulumi/pulumi v2.0.0.
```
Comment (justinvp) [inline CHANGELOG.md:14]:
```suggestion
- Use consistent naming of policies.
- Add rdsInstanceMultiAZEnabled policy.
- Upgrade to @pulumi/policy v1.1.0 and @pulumi/pulumi v2.0.0.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,14 +1,18 @@
...
 
-- Use consistent naming of policies.
-- Add rdsInstanceMultiAZEnabled policy.
-- Upgrade to @pulumi/policy v1.1.0 and @pulumi/pulumi v2.0.0.
+- Use consistent naming of policies.	
+- Add rdsInstanceMultiAZEnabled policy.	
+- Upgrade to @pulumi/policy v1.1.0 and @pulumi/pulumi v2.0.0.	
 
-## 0.1.0 (2019-11-26)
+## 0.1.0 (2019-11-26)
```
Comment (justinvp) [inline CHANGELOG.md:16]:
```suggestion
## 0.1.0 (2019-11-26)
```

File: CHANGELOG.md (hunk 4 above)
Comment (andrewpurdin) [reply CHANGELOG.md:10]: Sorry, not sure how that got deleted in the first place 😬

File: CHANGELOG.md (hunk 1 above)
Comment (andrewpurdin) [reply CHANGELOG.md:1]: I think this is ready to go now.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,13 +1,17 @@
...
+
+- Add root volume encryption validation to `encrypted-volumes` policy.
+
 ## 0.2.1 (2020-05-29)
 
 - Fix TypeScript compiler error when using the library (thanks @rsclarke-vgw!)
   [#60](https://github.com/pulumi/pulumi-policy-aws/pull/60)
 
-## 0.2.0 (2020-04-17)
+## 0.2.0 (2020-04-17)
```
Comment (justinvp) [inline CHANGELOG.md:10]:
```suggestion
## 0.2.0 (2020-04-17)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,13 +1,17 @@
...
 
-## 0.2.0 (2020-04-17)
+## 0.2.0 (2020-04-17)	
 
-- Use consistent naming of policies.
-- Add rdsInstanceMultiAZEnabled policy.
-- Upgrade to @pulumi/policy v1.1.0 and @pulumi/pulumi v2.0.0.
+- Use consistent naming of policies.	
+- Add rdsInstanceMultiAZEnabled policy.	
+- Upgrade to @pulumi/policy v1.1.0 and @pulumi/pulumi v2.0.0.
```
Comment (justinvp) [inline CHANGELOG.md:14]:
```suggestion
- Use consistent naming of policies.
- Add rdsInstanceMultiAZEnabled policy.
- Upgrade to @pulumi/policy v1.1.0 and @pulumi/pulumi v2.0.0.
```

**Pull request merged**